gridstatus/tests/test_logger.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+
+from gridstatus.logging import log
+
+
+@pytest.fixture
+def msg():
+    return "testing from test_logger.py"
+
+
+def test_log_stdout(msg, caplog):
+    log(msg, verbose=True)
```

Comment:
i think you need a test where verbose=False too

Reply:
Fixed:
https://github.com/kmax12/gridstatus/pull/153/commits/3d390236144a9e78e7619f3a7681352635044603